src/taoensso/sente.cljx

```diff
@@ -81,6 +81,16 @@
   (:require-macros
    [cljs.core.async.macros :as asyncm :refer (go go-loop)]))
 
+#+cljs
+(def is-node? (nil? enc/js-?win))
```

Comment:
That's correct, here's the docstring:

"Var bound to the name value of the compiler build :target option.
  For example, if the compiler build :target is :nodejs, *target* will be bound
  to \"nodejs\". *target* is a Google Closure define and can be set by compiler
  :closure-defines option."